configuration/consoleHelp.js

```diff
@@ -0,0 +1,43 @@
+const chalk = require("chalk");
```

Comment:
didn't you guys have webpack configured to convert ES7 to Node? Why are you using require?